modules/gitops-release-notes-1-5-7.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-5-7_{context}"]
+= Release notes for {gitops-title} 1.5.7
+
+{gitops-title} 1.5.7 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-5-7_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* From {product-title} 4.12, it will be optional to install the console. This fix updates the {gitops-title} Operator to prevent errors with the Operator if the console is not installed. link:https://issues.redhat.com/browse/GITOPS-2353[GITOPS-2353]
```

Comment:
For consistency, you want to grammatically construct each release note in the same way. Therefore, choose future tense or present tense. The IBM SG wants us to avoid future tense, and I think you can use present tense here. So I'd make the change to all the RNs, so that it's present tense, like:

`From {product-title} 4.12, it is optional to install the console.`